addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -1131,8 +1131,8 @@
 		
 		"595"	//The Manmelter
 		{
-			"desp"			"Manmelter: {positive}+10% damage and +10% firing speed on hit (capped at +200% damage and +88% firing speed)"
-			"attrib"		"2 ; 1.0 ; 6 ; 1.0"	//This is so the stats can be modified
+			"desp"			"Manmelter: {positive}+20% damage and +10% firing speed on hit (capped at +300% damage and +80% firing speed), {negative}crits to minicrits"
```

Comment:
`crit become minicrits` would be a better wording for it

Reply:
done